apps/ewallet_db/priv/repo/reporters/seeds.exs

```diff
@@ -1,6 +1,8 @@
 defmodule EWalletDB.Repo.Reporters.SeedsReporter do
+  alias EWalletConfig.Config
+
   def run(writer, args) do
-    base_url = Application.get_env(:ewallet_db, :base_url) || "https://example.com"
+    base_url = Config.get("base_url", "https://example.com")
```

Comment:
I couldn't find `Config.get/2` in `EWalletConfig.Config`.

Reply:
It's the main one -> https://github.com/omisego/ewallet/blob/6b30ffd209baab97211507f6b6b5455511c32f5f/apps/ewallet_config/lib/ewallet_config/config.ex#L121